x-pack/test/upgrade/apps/dashboard/dashboard_smoke_tests.ts

```diff
@@ -58,7 +59,11 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
           log.debug('Checking saved searches rendered');
           await dashboardExpect.savedSearchRowCount(49);
           log.debug('Checking input controls rendered');
-          await dashboardExpect.controlCount(3);
+          if (semver.lt(process.env.ORIGINAL_VERSION!, '8.5.0-SNAPSHOT')) {
```

Comment:
I'll close my PR which I created with the exact same code as this.  The only difference is that I thought the controls were changed in 8.6.0 (main branch), not 8.5.0.  Maybe @ThomThomson could confirm?

Reply:
I was not sure that is why I was running the tests I have the results now and yes it appears it needs it from 8.6.0 I will change it.